tests/wpt/web-platform-tests/url/a-element-href-javascript.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<meta charset=utf-8>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<script>
+var linkStatus = 'link not clicked';
+
+function changeStatus() {
+  linkStatus = 'link has been clicked';
+}
+</script>
+
+<a id="javascript-link" href="javascript:changeStatus()">link</a>
+
+<script>
+setup({explicit_done:true});
+
+document.querySelector("#javascript-link").click();
+
+step_timeout(function() {
```

Comment:
Let's do this instead:
```html
<script>
function changeStatus() {
  t.done();
}

var t = async_test(function(t) {
  document.querySelector("#javascript-link").click();
}, "javascript: scheme urls should be executed in current global scope");
</script>
```

Reply:
Fixed in https://github.com/servo/servo/pull/17083/commits/fc23cb1a6306e830e622b93e6eccc13cf66bd0a4